package.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
appwrite package is a frontend dependency so should be in frontends package.json
Also use yarn instead of npm to maintain proper consistency